evap/evaluation/views.py

```diff
@@ -94,3 +102,27 @@ def faq(request):
 
 def legal_notice(request):
     return render(request, "legal_notice.html", dict())
+
+
+@require_POST
+@login_required
+def feedback_send(request):
+    sender_email = request.POST.get("sender_email")
+    message = request.POST.get("message")
+    subject = "Feedback from {}".format(sender_email)
+
+    if message:
+        mail = EmailMessage(
+            from_email=sender_email,
+            subject=subject,
+            body=message,
+            to=[settings.FEEDBACK_EMAIL])
+
+        try:
+            mail.send(False)
+            logger.info(('Sent email "{}"').format(mail.message()))
```

Comment:
It looks something like this

Content-Type: text/b; charset="us-ascii"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: subj
From: asdf@asdf.de

body stuff here